lib/Jsrt/JsrtDiag.cpp

```diff
@@ -658,14 +658,15 @@ CHAKRA_API JsDiagEvaluate(
             return JsErrorInvalidArgument;
         }
 
-        Js::DynamicObject* result = debuggerStackFrame->Evaluate(scriptContext, expression, static_cast<int>(len), false);
+        Js::DynamicObject* result = nullptr;
```

Comment:
nit: you don't need to use this extra variable *result*, as after the Evaluate api, if the result is null then *evalResult is already be null.